packages/fast-usdc-contract/test/mocks.ts

```diff
@@ -127,7 +127,7 @@ export const makeTestLogger = (logger: LogFn) => {
     logger(args);
   };
   const inspectLogs = (index?: number) =>
```

Comment:
consider typing the return type, which is conditioned on `index` being provided.